packages/components/src/components/wallet-cfd-success-dialog/wallet-cfd-card.scss

```diff
@@ -0,0 +1,73 @@
+.wallet-cfd-card {
+    width: 24rem;
+    height: 14.4rem;
+    position: relative;
+    overflow: hidden;
+    border-radius: $BORDER-RADIUS;
+    background: var(--wallets-card-active-gradient-background);
+
+    &__icon {
+        position: absolute;
+        top: 16px;
+        left: 16px;
+    }
+
+    &__cfd-icon {
+        position: relative;
+        z-index: 1;
+    }
+
+    &__wallet-icon {
+        position: absolute;
+        top: 12px;
+        left: 12px;
```

Comment:
```suggestion
        left: 1.2rem;
```

Reply:
Isn't that relative units? I am using `position: absolute`  to make the icon to be in that specific position hence why I was using `px`. Correct me if I am wrong @mahdiyeh-deriv

Reply:
yes

Reply:
@aizad-deriv the rem is relative to the default font-size and since our default font0size is 10, 1rem will not change in order to the place you are using that.